sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeEntitiesTests.cs

```diff
@@ -38,47 +38,33 @@ public class RecognizeEntitiesTests : TextAnalyticsClientLiveTestBase
         public async Task RecognizeEntitiesWithAADTest()
         {
             TextAnalyticsClient client = GetClient(useTokenCredential: true);
-            string document = singleEnglish;
+            string document = SingleEnglish;
 
             CategorizedEntityCollection entities = await client.RecognizeEntitiesAsync(document);
 
-            Assert.AreEqual(3, entities.Count);
-
-            var entitiesList = new List<string> { "Bill Gates", "Microsoft", "Paul Allen" };
-            foreach (CategorizedEntity entity in entities)
-            {
-                Assert.IsTrue(entitiesList.Contains(entity.Text));
-                Assert.IsNotNull(entity.ConfidenceScore);
-            }
+            ValidateInDocumenResult(entities, 3);
         }
 
         [Test]
         public async Task RecognizeEntitiesTest()
         {
             TextAnalyticsClient client = GetClient();
-            string document = singleEnglish;
+            string document = SingleEnglish;
 
             CategorizedEntityCollection entities = await client.RecognizeEntitiesAsync(document);
 
-            Assert.AreEqual(3, entities.Count);
-
-            var entitiesList = new List<string> { "Bill Gates", "Microsoft", "Paul Allen" };
-            foreach (CategorizedEntity entity in entities)
-            {
-                Assert.IsTrue(entitiesList.Contains(entity.Text));
```

Comment:
It doesn't look like the refactored method actually tests that the expected text was found. Consider passing that text into the validator method.

Reply:
I was thinking on leaving it out as we are testing our SDK and not necessarily the service.
In the past, we've had our test broken when the service updates their models so the output changes. We for sure expect at least something, but  not care about what it is.
What do you think?

Reply:
Would it make sense to test that you are at least getting one of the words you expect in the response from the service? That way we can at least verify that the service is doing something reasonable.